docs/janssen-server/developer/agama/quick-start-using-agama-lab.md

```diff
@@ -285,80 +342,94 @@ Use the steps below to create the page.
-           }
-       </style>
+        <!DOCTYPE html>
+    <html lang="en">
+        <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Username_Password_Form
+      </title>
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/tailwindcss@2.2/dist/tailwind.min.css" type="text/css">
+      </head>
+      <body>
+      <div class="md:h-screen bg-white relative flex flex-col justify-center items-center">
+        <div class="md:border md:border-gray-300 bg-white md:shadow-lg shadow-none rounded p-10 align-middle">
+        <div class="flex flex-col items-center space-y-3">
+          <div class="flex">
+          <img src="https://gluu.org/wp-content/uploads/2020/12/logo.png" id="i4xwu">
+          </div>
+          <span class="text-2xl font-semi-bold leading-normal">Form</span>
+          <p class="leading-normal">Enter your Username and Password
+          </p>
+        </div>
+        <form method="post" class="my-8">
+          <div class="relative mb-2">
+          <label for="username" class="label mt-2 ml-1 leading-tighter text-gray-600 text-base cursor-text">
+            Username
+          </label>
+          <input type="text" id="username" name="username" autofocus="" required="" class="w-full rounded px-3 border border-gray-300 pt-2 pb-2 focus:border-green-700">
+          </div>
+          <div class="relative mb-2">
+          <label for="password" class="label mt-2 ml-1 leading-tighter text-gray-600 text-base cursor-text">
+            Password
+          </label>
+          <input type="password" id="password" name="password" autofocus="" required="" class="w-full rounded px-3 border border-gray-300 pt-2 pb-2 focus:border-green-700">
+          </div>
+          <div class="space-y-9">
+          <div class="text-sm flex justify-center items-center">
+            <button type="submit" class="py-2 px-6 rounded text-white btn bg-green-500 hover:bg-green-600">
+            Submit
+            </button>
+          </div>
+          </div>
+        </form>
+        </div>
+      </div>
+      </body>
     </html>
     ```
 
-    _Click `Save changes`_
+_Click `Save changes`_
 
-    ![](../../../assets/agama-lab-flow-passwd-save-template.png)
+![](../../../assets/agama-lab-flow-save-template.png)
 
-    _This will render the page in the visual editor_.
+_This will render the page in the visual editor_.
 
-    ![](../../../assets/agama-lab-flow-passwd-render-template.png)
+![](../../../assets/agama-lab-flow-render-template.png)
 
-### Release The Project
+### Release the project
 
-To use the flow for authentication in the Janssen Server, the flow needs to be released. Agama Lab
-releases the flow and the related artifacts (like template) in the form of a `.gama` file in the GitHub repository.
+At this point we have all the components of an Agama project ready, and they are 
```

Comment:
As the screenshot shows, all Agama stuff should be on the main branch to create a release. So there should be instructions added to make a PR from `agama-lab-policy-designer` branch to default branch.